# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/validationGL3_autogen.h
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/libANGLE/validationGL3_autogen.h
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/angle/checkout/src/libANGLE/validationGL3_autogen.h	Sat Sep 07 13:38:36 2019 +0000
@@ -0,0 +1,83 @@
+// GENERATED FILE - DO NOT EDIT.
+// Generated by generate_entry_points.py using data from gl.xml and wgl.xml.
+//
+// Copyright 2019 The ANGLE Project Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+//
+// validationGL3_autogen.h:
+//   Validation functions for the OpenGL 3.0 entry points.
+
+#ifndef LIBANGLE_VALIDATION_GL3_AUTOGEN_H_
+#define LIBANGLE_VALIDATION_GL3_AUTOGEN_H_
+
+#include "common/PackedEnums.h"
+
+namespace gl
+{
+class Context;
+
+bool ValidateBeginConditionalRender(Context *context, GLuint id, GLenum mode);
+bool ValidateBindFragDataLocation(Context *context,
+                                  GLuint program,
+                                  GLuint color,
+                                  const GLchar *name);
+bool ValidateClampColor(Context *context, GLenum target, GLenum clamp);
+bool ValidateColorMaski(Context *context,
+                        GLuint index,
+                        GLboolean r,
+                        GLboolean g,
+                        GLboolean b,
+                        GLboolean a);
+bool ValidateDisablei(Context *context, GLenum target, GLuint index);
+bool ValidateEnablei(Context *context, GLenum target, GLuint index);
+bool ValidateEndConditionalRender(Context *context);
+bool ValidateFramebufferTexture1D(Context *context,
+                                  GLenum target,
+                                  GLenum attachment,
+                                  GLenum textarget,
+                                  GLuint texture,
+                                  GLint level);
+bool ValidateFramebufferTexture3D(Context *context,
+                                  GLenum target,
+                                  GLenum attachment,
+                                  TextureTarget textargetPacked,
+                                  GLuint texture,
+                                  GLint level,
+                                  GLint zoffset);
+bool ValidateGetTexParameterIiv(Context *context,
+                                TextureType targetPacked,
+                                GLenum pname,
+                                GLint *params);
+bool ValidateGetTexParameterIuiv(Context *context,
+                                 TextureType targetPacked,
+                                 GLenum pname,
+                                 GLuint *params);
+bool ValidateIsEnabledi(Context *context, GLenum target, GLuint index);
+bool ValidateTexParameterIiv(Context *context,
+                             TextureType targetPacked,
+                             GLenum pname,
+                             const GLint *params);
+bool ValidateTexParameterIuiv(Context *context,
+                              TextureType targetPacked,
+                              GLenum pname,
+                              const GLuint *params);
+bool ValidateVertexAttribI1i(Context *context, GLuint index, GLint x);
+bool ValidateVertexAttribI1iv(Context *context, GLuint index, const GLint *v);
+bool ValidateVertexAttribI1ui(Context *context, GLuint index, GLuint x);
+bool ValidateVertexAttribI1uiv(Context *context, GLuint index, const GLuint *v);
+bool ValidateVertexAttribI2i(Context *context, GLuint index, GLint x, GLint y);
+bool ValidateVertexAttribI2iv(Context *context, GLuint index, const GLint *v);
+bool ValidateVertexAttribI2ui(Context *context, GLuint index, GLuint x, GLuint y);
+bool ValidateVertexAttribI2uiv(Context *context, GLuint index, const GLuint *v);
+bool ValidateVertexAttribI3i(Context *context, GLuint index, GLint x, GLint y, GLint z);
+bool ValidateVertexAttribI3iv(Context *context, GLuint index, const GLint *v);
+bool ValidateVertexAttribI3ui(Context *context, GLuint index, GLuint x, GLuint y, GLuint z);
+bool ValidateVertexAttribI3uiv(Context *context, GLuint index, const GLuint *v);
+bool ValidateVertexAttribI4bv(Context *context, GLuint index, const GLbyte *v);
+bool ValidateVertexAttribI4sv(Context *context, GLuint index, const GLshort *v);
+bool ValidateVertexAttribI4ubv(Context *context, GLuint index, const GLubyte *v);
+bool ValidateVertexAttribI4usv(Context *context, GLuint index, const GLushort *v);
+}  // namespace gl
+
+#endif  // LIBANGLE_VALIDATION_GL3_AUTOGEN_H_